### DISCUSSION TITLE
Enabling independent version management for subprojects

### DISCUSSION THREAD

**Discussion by RODRIGOOXFI** [dipien/semantic-version-gradle-plugin#51] [Ideas]
Hi guys, 

I'm currently working in a project where is required to implement automatic semantic versioning into some specific projects. These projects are in a single repository but it is required to have independent versions even if everybody can push to the same repo for different projects.

**Describe the solution you'd like**
Is there a way to implement the plugin but keeping independent version bumping between subprojects? It'will be great to tag versions (prereleases and releases) with custom suffix to keep track of the version changes per project.

I'm gonna try to fork the plugin and implement something by my self. I'll keep you guys updated in case that something good happens, but if you have any insight on how to use this plugin in something similar to what I'm looking also will be great.

Thanks a lot

**Comment by maxirosson** [comment]
Are all the projects part of the same Gradle project? If you have different projects on the same repo not tied to the same version, maybe having one gradle project per project could be a good idea.
